### ISSUE TITLE
No issue!

### ISSUE THREAD

**Issue by MrKusty** [sxlar3d/Overwatch-Ik-Rig-Blender-Addon#1]
No issue at all! just can't wait for future update :)

**Comment by sxlar3d** [comment]
> 

yep ive been a bit unmotivated but once im done with this art piece im making ill be back at it!
